src/seedu/addressbook/commands/HistoryCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+
+import java.util.Stack;
+
+import static seedu.addressbook.Main.CommandHistory;
+
+/**
+ * Displays all the past commands inputted by the user.
+ */
+public class HistoryCommand extends Command{
```

Comment:
Space before the brace!

Reply:
Oh thanks for pointing out! Changed now.